stdlib/pair.mli

```diff
@@ -0,0 +1,66 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                         The OCaml programmers                          *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Operations on pairs.
+
+  @since 5.3 *)
```

Comment:
```suggestion
  @since 5.4 *)
```

Reply:
Ok but why? 5.3 isn't released yet right?

Reply:
It's been branched, and is in feature-freeze.

Reply:
Alright then ;) commited, thanks